portable/ARMv8M/non_secure/portable/GCC/ARM_CM23/portmacro.h

```diff
@@ -60,6 +60,14 @@
 #if ( configTOTAL_MPU_REGIONS == 16 )
     #error 16 MPU regions are not yet supported for this port.
 #endif
+
+#ifndef configENABLE_MVE
+    #define configENABLE_MVE    0
+#else
+    #if( configENABLE_MVE != 0 )
+        #error configENABLE_MVE must be 0 for the Cortex-M23.
```

Comment:
Must be left undefined, or defined to 0.  Same comment applies in multiple places.

Reply:
Done.